internal/pkg/template/templates/workloads/partials/cf/imported-alb-resources.yml

```
@@ -0,0 +1,151 @@
+# If there's an HTTPS listener on the imported ALB, forward traffic to tasks.
+{{$httpsARN := .ImportedALB.HTTPSListenerARN}}
+{{- if ne $httpsARN ""}}
+HTTPSRulePriorityAction:
+  Metadata:
+    'aws:copilot:description': 'A custom resource assigning priority for HTTPS listener rules'
+  Type: Custom::RulePriorityFunction
+  Properties:
+    ServiceToken: !GetAtt RulePriorityFunction.Arn
+    RulePath: {{ fmtSlice (quoteSlice .ALBListener.RulePaths) }}
+    ListenerArn: {{$httpsARN}}
+
+{{- range $i, $rule := .ALBListener.Rules}}
+HTTPSListenerRuleForImportedALB{{ if ne $i 0 }}{{ $i }}{{ end }}:
+  Metadata:
+    'aws:copilot:description': 'An HTTPS listener rule for path `{{$rule.Path}}` that forwards HTTPS traffic to your tasks'
+  Type: AWS::ElasticLoadBalancingV2::ListenerRule
+  Properties:
+    Actions:
+      - TargetGroupArn: !Ref TargetGroup{{ if ne $i 0 }}{{ $i }}{{ end }}
+        Type: forward
+    Conditions:
+      {{- if $rule.AllowedSourceIps}}
+      - Field: 'source-ip'
+        SourceIpConfig:
+          Values:
+          {{- range $sourceIP := $rule.AllowedSourceIps}}
+          - {{$sourceIP}}
+          {{- end}}
+      {{- end}}
+      {{- if $rule.Aliases}}
+      - Field: 'host-header'
+        HostHeaderConfig:
+          Values: {{fmtSlice (quoteSlice $rule.Aliases)}}
+      {{- end}}
+      - Field: 'path-pattern'
+        PathPatternConfig:
+          Values:
+            {{- if eq $rule.Path "/" }}
+            - /*
+            {{- else }}
+            - {{ $rule.Path }}
+            - {{ $rule.Path }}/*
+            {{- end }}
+    ListenerArn: {{$httpsARN}}
+    Priority: !GetAtt HTTPSRulePriorityAction.Priority{{ if ne $i 0 }}{{ $i }}{{ end }}
+  {{- end}}
```

Can we add comments for end statement like below for this file and fix indentation 🥺 
```suggestion
  {{- end}}{{/* endrange $i, $rule := .ALBListener.Rules */}}
```